test_scripts/Polices/Policies_Security/001_ATF_P_PolicyTable_GeneralSDL_behavior.lua

```diff
@@ -0,0 +1,82 @@
+----  Name of requirement that is covered.
+----- [General] Policies enough memory for PTS
+----  Policies Manager must have enough memory to hold a Policy Table Snapshot in memory.
+
+--------------------------------------------------------------------------------------------
+-----------------------------Required Shared Libraries---------------------------------------
+---------------------------------------------------------------------------------------------
+local mobile_session = require('mobile_session')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+config.defaultProtocolVersion = 2
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Local preparing ]]
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ Preconditions ]]
+commonFunctions:userPrint(33, "================= Precondition ==================")
+Test = require('connecttest')
+
+--[[Test Case]]
+function Test:GeneralSDLbehaviour_PTsize_1mb()
```

Comment:
@Aleksandriys verification criteria from your requirement:
**Before sending PTS via OnSystemRequest to mobile app, SDL checks to have available more than 1 Mbytes of allocated memory RAM.**
This check is not done in script at all!!